app/views/citizens/check_answers/_bank_accounts.html.erb

```diff
@@ -24,5 +24,5 @@
 <% end %>
 
 <div class="govuk-body govuk-!-padding-bottom-4">
-  <%= link_to_accessible t("citizens.check_answers.index.link_other_account"), citizens_banks_path, class: "govuk-link" %>
+  <%= govuk_link_to t("citizens.check_answers.index.link_other_account"), citizens_banks_path, class: "govuk-link" %>
```

Comment:
I think the class `govuk-link` can be removed as it is applied by default by the helper `govuk_link_to`

Reply:
Thank you for the tip. I have made these suggested changes now.